crates/jstz_api/tests/wpt.rs

```diff
@@ -282,6 +282,9 @@ pub fn run_wpt_test_harness(bundle: &Bundle) -> JsResult<Box<TestHarnessReport>>
         }
     }
 
+    // Execute promises after all sync tests have completed after `eval` returns
+    rt.run_jobs();
+
```

Comment:
Prefer our event loop over Boa's:
```suggestion
    // Execute promises after all sync tests have completed after `eval` returns
    rt.run_event_loop()

```

Reply:
I tried this but `rt.run_event_loop().await` simply hanged there. I'm not sure if this has anything to do with the async context blocking itself or so. Do you have any idea? If not, I can look into this later.

Reply:
I'm not too sure why that would be the case -- it is likely something odd with async since `run_event_loop` essentially loops `poll_event_loop` until `Poll::Ready(())` is returned. I just noticed that we override the default implementation of `run_jobs` for our job queue so `rt.run_jobs()` should fine anyway